src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/HelperMarshal.cs

```diff
@@ -66,6 +66,12 @@ private static void InvokeString2(string s)
             _stringResource2 = s;
         }
 
+        private static string Issue48345(string s)
+        {
+            _stringResource = $"s: {s} length: {s?.Length}";
```

Comment:
Nit: This has null check on the `s` variable but do not see a test for `null`.

Reply:
Doesn't need one, string format on a null will just be ""

Reply:
👍